### DISCUSSION TITLE
Is there any way to pre-fill File Finder text field via URL?

### DISCUSSION THREAD

**Discussion by nbahhir** [community/community#37350] [New to GitHub]
I tried baseURL/?query=test and baseURL/?tree-finder-field=test, but the text field keep resetting.

**Comment by ossanna16** [comment]
Hi @nbahhir,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on web development. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
